### ISSUE TITLE
module 'rdkit.Chem.Draw.rdMolDraw2D' has no attribute 'PrepareAndDrawMolecule'

### ISSUE THREAD

**Issue by LanceKnight** [rdkit/rdkit#3686]
**Describe the bug**
rdkit.Chem.Draw.rdMolDraw2D has no attribute "PrepareAndDrawMolecule"

**To Reproduce**
`from rdkit.Chem.Draw import rdMolDraw2D
from rdkit import Chem
smi = 'c1cc(F)ccc1Cl'
mol = Chem.MolFromSmiles(smi)
patt = Chem.MolFromSmarts('ClccccF')
hit_ats = list(mol.GetSubstructMatch(patt))
hit_bonds = []
for bond in patt.GetBonds():
    aid1 = hit_ats[bond.GetBeginAtomIdx()]
    aid2 = hit_ats[bond.GetEndAtomIdx()]
    hit_bonds.append(mol.GetBondBetweenAtoms(aid1,aid2).GetIdx())
    d = rdMolDraw2D.MolDraw2DSVG(500, 500) # or MolDraw2DCairo to get PNGs
    rdMolDraw2D.PrepareAndDrawMolecule(d, mol, highlightAtoms=hit_ats, highlightBonds=hit_bonds)`

**Expected behavior**
rdMolDraw2D should have this attribute

**Configuration (please complete the following information):**
 - RDKit version: '2017.09.1'
 - OS: [Windows Anaconda3]
 - Python version (if relevant): 3.6.12
 - Are you using conda? Yes
 - If you are using conda, which channel did you install the rdkit from? channel rdkit
 - If you are not using conda: how did you install the RDKit? NA

**Additional context**
These codes are from https://www.rdkit.org/docs/GettingStartedInPython.html#drawing-molecules

**Comment by mattwthompson** [comment]
That version of RDKit is several updates old, you should update to a more recent version (2020.x.x)

**Comment by LanceKnight** [comment]
I see. I used the command from the rdkit installation page: https://www.rdkit.org/docs/Install.html and I thought that gave me the latest version. Maybe that page needs to be updated too :)

**Comment by greglandrum** [comment]
I believe that you should get the most recent version if you execute those instructions as they are provided.
If you try to install the RDKit into an existing environment which has an older version of one of the dependencies, then conda will do the "right thing" and give you an older version of the RDKit.

**Comment by LanceKnight** [comment]
I just did an experiment with the command on the website and it seems like I still get the old version. To avoid using an existing environment, I deleted old environment ( I also checked D:/Anaconda3/envs and that environment was indeed deleted) and entered `conda create -c rdkit -n my-rdkit-env rdkit `and below is the output:

```
done
#
# To activate this environment, use
#
#     $ conda activate my-rdkit-env
#
# To deactivate an active environment, use
#
#     $ conda deactivate


C:\Users\liuy69>conda activate my-rdkit-env

(my-rdkit-env) C:\Users\liuy69>conda list
# packages in environment at D:\Anaconda3\envs\my-rdkit-env:
#
# Name                    Version                   Build  Channel
blas                      1.0                         mkl
boost                     1.59.0                   py36_3    rdkit
bzip2                     1.0.8                he774522_0
certifi                   2020.12.5        py36haa95532_0
freetype                  2.10.4               hd328e21_0
intel-openmp              2020.2                      254
jpeg                      9b                   hb83a4c4_2
libpng                    1.6.37               h2a8f88b_0
libtiff                   4.1.0                h56a325e_1
lz4-c                     1.9.2                hf4a77e7_3
mkl                       2020.2                      256
mkl-service               2.3.0            py36h196d8e1_0
mkl_fft                   1.2.0            py36h45dec08_0
mkl_random                1.1.1            py36h47e9c7a_0
numpy                     1.19.2           py36hadc3359_0
numpy-base                1.19.2           py36ha3acd2a_0
olefile                   0.46                     py36_0
pandas                    1.1.5            py36hd77b12b_0
pillow                    8.0.1            py36h4fa10fc_0
pip                       20.3.3           py36haa95532_0
python                    3.6.12               h5500b2f_2
python-dateutil           2.8.1                      py_0
pytz                      2020.5             pyhd3eb1b0_0
rdkit                     2017.09.1                py36_1    rdkit
setuptools                51.0.0           py36haa95532_2
six                       1.15.0           py36haa95532_0
sqlite                    3.33.0               h2a8f88b_0
tk                        8.6.10               he774522_0
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wheel                     0.36.2             pyhd3eb1b0_0
wincertstore              0.2              py36h7fe50ca_0
xz                        5.2.5                h62dcd97_0
zlib                      1.2.11               h62dcd97_4
zstd                      1.4.5                h04227a9_0

(my-rdkit-env) C:\Users\liuy69>     
```

**Comment by greglandrum** [comment]
Ah, ok, maybe this is a windows thing. I will take a look

**Comment by LanceKnight** [comment]
Sounds good. Thank you!

**Comment by LanceKnight** [comment]
I checked Programs and Features and it says I installed it on 11/16/2020. I bought the laptop in 2019 so it won't be several years ago.

**Comment by IchiruTake** [comment]
> **Describe the bug**
> rdkit.Chem.Draw.rdMolDraw2D has no attribute "PrepareAndDrawMolecule"
> 
> **To Reproduce**
> `from rdkit.Chem.Draw import rdMolDraw2D from rdkit import Chem smi = 'c1cc(F)ccc1Cl' mol = Chem.MolFromSmiles(smi) patt = Chem.MolFromSmarts('ClccccF') hit_ats = list(mol.GetSubstructMatch(patt)) hit_bonds = [] for bond in patt.GetBonds(): aid1 = hit_ats[bond.GetBeginAtomIdx()] aid2 = hit_ats[bond.GetEndAtomIdx()] hit_bonds.append(mol.GetBondBetweenAtoms(aid1,aid2).GetIdx()) d = rdMolDraw2D.MolDraw2DSVG(500, 500) # or MolDraw2DCairo to get PNGs rdMolDraw2D.PrepareAndDrawMolecule(d, mol, highlightAtoms=hit_ats, highlightBonds=hit_bonds)`
> 
> **Expected behavior**
> rdMolDraw2D should have this attribute
> 
> **Configuration (please complete the following information):**
> 
> * RDKit version: '2017.09.1'
> * OS: [Windows Anaconda3]
> * Python version (if relevant): 3.6.12
> * Are you using conda? Yes
> * If you are using conda, which channel did you install the rdkit from? channel rdkit
> * If you are not using conda: how did you install the RDKit? NA
> 
> **Additional context**
> These codes are from https://www.rdkit.org/docs/GettingStartedInPython.html#drawing-molecules

you can use `conda create -c rdkit -n my-rdkit-env rdkit==2020.09.1 python==3.7.9 `

**Comment by greglandrum** [comment]
So when I try `conda create -c rdkit -n rdkit_test rdkit` on my windows machine I get v2020.09.1 of the rdkit.
That's python3.7 for me. If I force python 3.6 with `conda create -c rdkit -n rdkit_test rdkit python=3.6` I also get v2020.09.1
 
This is using conda v4.9.2. Could it be that you are using an older version of conda?

**Comment by LanceKnight** [comment]
I checked my conda and it is the latest version. But since it works for you, it might just be some setting on my computer that causes the problem.

```
(base) C:\Users\liuy69>conda -V
conda 4.9.2
```

Hi, @IchiruTake, I did get the latest version with the equal symbol for my own use. I was just concerning other people might get old version without knowing. But it seems like it is still just my own computer's problem. Thank you for the suggestion anyway:)
